RELEASE.md

```diff
@@ -0,0 +1,27 @@
+# Hdclient Release
+
+This document contains the steps to be followed to release
+the project hdclient.
+
+## Release Process
+
+To release hdclient:
+
+* Create a new branch: `git checkout -b feature/TICKET-123-release-x.y.z`
+
+* Name the tag for the repository.
+
+* Use the `npm version $tag` command with the same tag to update [package.json].
+
+* Create a PR and merge the [package.json] change.
+
+* Checkout the change in the development branch:
+  1. `git checkout development/x.y`
+  1. `git pull`
+
+* Tag the repository using the same tag:
+  1. `git tag --annotate $tag`
+  1. `git push origin $tag`
+
+
+[package.json]: ./package.json
```

Comment:
should we document the eve steps too ? 
As far as I know we need to promote the build once the tag is set.

Reply:
right.